scudcloud-1.0/lib/wrapper.py

```diff
@@ -106,6 +106,12 @@ def _linkClicked(self, qUrl):
         else:
             self.systemOpen(url)
 
+    def gotoNextOpenChannelOrIM(self, unreads_only=False, backwards=False):
+        return self.page().currentFrame().evaluateJavaScript(
+            'TS.client.ui.gotoNextOpenChannelOrIM(%s, %s);' % (
+                str(bool(unreads_only)).lower(),
+                str(bool(backwards)).lower()))
+
```

Comment:
Ideally, we should change `call` to accept dynamic arguments... but I think we can live with that for now.